### ISSUE TITLE
File "ph.py", line 6, in <module> from pushetta import Pushetta

### ISSUE THREAD

**Issue by adelorenzo** [IvanTurgenev/whocasa#2]
Hello,

I went through all the requirements and installed pushetta via pip3 successfully on my raspberry pi. When I try to run the ph.py I get the error message:

File "ph.py", line 6, in <module> 
from pushetta import Pushetta

What could have gone wrong?

Thanks

**Comment by IvanTurgenev** [comment]
Did you install pushetta? sudo pip3 install pushetta

**Comment by adelorenzo** [comment]
Yes, on both Pi's that I own. Coincidently both are running raspbian Jessie lite. Could that be the issue?

**Comment by macmuchmore** [comment]
Did you ever figure this out?
